### DISCUSSION TITLE
About tile's Down Sampling Rate

### DISCUSSION THREAD

**Discussion by xiaoxuwei12345** [Mikubill/sd-webui-controlnet#1479] [General]
In the webui of the tile model, there is the parameter Down Sampling Rate, but it is not in the introduction(https://github.com/Mikubill/sd-webui-controlnet/wiki/API).  What is the writing method of this parameter? "down_sampling_rate"?

**Comment by WhiteMemory99** [comment]
You should use the "threshold_a" parameter, if I understand the API correctly.
